### PR TITLE
RTC-10806 - Smaller UI improvements on the Dropdown/Select component

### PR DESCRIPTION
**Pull request by axeleriksson147**
https://perzoinc.atlassian.net/browse/RTC-10806

- Aligned the Select component with the SDS (Symphony Design System) - https://www.figma.com/file/V0VKw74594EyCKHwqYk6LV/%E2%9D%96-SDS-Core-Component-Library?node-id=5453%3A17120
- - triangle to select should be a different grey
- - font should be 14px

- Changed story font to "system-ui", removed toEm() and formatting of Select story.

### CODE REVIEW

File: src/uitoolkit-components/_select.scss [hunk 1]
```diff
@@ -4,9 +4,10 @@
   box-shadow: inset 0 0 0 toRem(2) $--tk-input-default-focus-border-color;
 }
 .tk-select {
+  font-size: toRem(14);
   .tk-select {
     &__single-value {
-      @extend .tk-text-color;
+      color: getColor($graphite, '50');
```
Comment (axeleriksson147) [inline src/uitoolkit-components/_select.scss:10]: https://www.figma.com/file/V0VKw74594EyCKHwqYk6LV/%E2%9D%96-SDS-Core-Component-Library?node-id=5453%3A17120

Comment (axeleriksson147) [inline src/uitoolkit-components/_select.scss:10]: https://www.figma.com/file/V0VKw74594EyCKHwqYk6LV/%E2%9D%96-SDS-Core-Component-Library?node-id=5453%3A17120 - Select component in SDS.

File: src/utils/_font.scss [hunk 2]
```diff
@@ -1,299 +1 @@
...
-      format('woff2'),
-    url('https://cdn.symphony.com/resources/api/v2.1/fonts/lato/Lato-BlackItalic.woff')
-      format('woff'),
-    url('https://cdn.symphony.com/resources/api/v2.1/fonts/lato/Lato-BlackItalic.ttf')
-      format('truetype');
-  font-style: italic;
-  font-weight: 900;
-  text-rendering: optimizeLegibility;
-}
+@charset "UTF-8";
```
Comment (axeleriksson147) [inline src/utils/_font.scss:1]: The components in SDS use "Segoe UI" - which is "system ui" as I understand it:

- "Segoe UI" for Windows
- "San Francisco" for Mac

This is the intended font for these components. We don't provide a font with toolkit but our examples mimic them.

File: src/utils/functions/index.scss [hunk 3]
```diff
@@ -1,9 +1,5 @@
 $UITOOLKIT_REM_BASE: 16 !default;
 
-@function toEm($size, $base: 14) {
```
Comment (axeleriksson147) [inline src/utils/functions/index.scss:3]: Not used anymore.

File: stories/fields.stories.js [hunk 4]
```diff
@@ -87,21 +87,10 @@ export const TextField = () => {
...
-<h3>Default</h3>
-<div>
-  <select type="text" class="tk-select" value="Text" style="width:200px">
-    <option>Item 1</option>
-    <option>Item 2</option>
-    <option>Item 3 with a very very very long text</option>
-  </select>
-  <i></i>
-</div>
+  <h3>Default</h3>
```
Comment (axeleriksson147) [inline stories/fields.stories.js:92]: Small formatting.

File: src/uitoolkit-components/_select.scss [hunk 5]
```diff
@@ -151,17 +152,17 @@
     }
     .tk-select__indicator {
       cursor: pointer;
-      color: inherit;
+      color: getColor($graphite, '50');
```
Comment (symphony-annaLlorens) [inline src/uitoolkit-components/_select.scss:155]: Is it the same for the dark mode?

Comment (axeleriksson147) [reply src/uitoolkit-components/_select.scss:155]: Yes, see here - https://www.figma.com/file/V0VKw74594EyCKHwqYk6LV/%E2%9D%96-SDS-Core-Component-Library?node-id=5453%3A17120

**Review by symphony-annaLlorens — APPROVED** [review]
LGTM 👍

File: src/uitoolkit-components/_select.scss (hunk 1 above)
Comment (symphony-adnane) [reply src/uitoolkit-components/_select.scss:10]: This won't react to dark theme

File: src/utils/_font.scss (hunk 2 above)
Comment (symphony-adnane) [reply src/utils/_font.scss:1]: These fonts being removed are probably being used by something else, 
For now I recommend not removing them until we check nobody's using them. 
In any case these are just the definitions and they aren't loaded unless they are requested to be used

**Review by symphony-adnane — APPROVED** [review]

**Pull request merged**